### PR TITLE
k8s.gcr.io: policy for what qualifies for a staging repo

### PR DESCRIPTION
**Pull request by spiffxp**
Related:
- Part of: https://github.com/kubernetes/k8s.io/issues/915

### CODE REVIEW

**Comment by spiffxp** [comment]
/assign @thockin @ameukam @dims 
/hold
for comment

File: k8s.gcr.io/README.md [hunk 1]
```diff
@@ -10,13 +10,31 @@ repositories used to publish official container images for Kubernetes.
...
+the main Kubernetes image-serving system (k8s.gcr.io).
+
+Access to each staging project is governed by a googlegroup, which grants the
+ability to manually trigger GCB or push container images in the event that
+automated builds via something like prow.k8s.io are not setup or are broken.
+
+### Requirements
+
+The rule of thumb is that staging repositories should be used to host
+artifacts produced by code that is part of the Kubernetes project. In other
```
Comment (ameukam) [inline k8s.gcr.io/README.md:25]:
```suggestion
artifacts produced by code that is part of the Kubernetes community. In other
```

Forgive my poor english but I feel like _Kubernetes project_ don't cover subprojects.

Comment (spiffxp) [reply k8s.gcr.io/README.md:25]: I have a problem with it the other way, where community could imply the broader ecosystem of projects and users that depend upon Kubernetes.  I used "project" here in the same context as "Kubernetes as a project is owned by the CNCF".  That covers all of the subprojects that are part of kubernetes.

I could capitalize Project to make it clearer I'm trying to cover a very specific definition here?

Comment (ameukam) [reply k8s.gcr.io/README.md]: +1 for _Kubernetes Project_.

Comment (spiffxp) [reply k8s.gcr.io/README.md]: I updated to define as present in one of the project-owned github organizations

File: k8s.gcr.io/README.md [hunk 2]
```diff
@@ -79,7 +99,7 @@ To promote an image, follow these steps:
 1. Merge the PR. Your image will be promoted by one of two jobs:
    - [`post-k8sio-image-promo`][post-promo-job] is a postsubmit that runs immediately after merge
    - [`ci-k8sio-cip`][ci-promo-job] is a postsubmit that runs immediately after merge
-1. A periodic 
+1. A periodic
```
Comment (thockin) [inline k8s.gcr.io/README.md:102]: I don't understand this fragment?

Comment (spiffxp) [reply k8s.gcr.io/README.md]: It was like this when I got here!  Looks like fixing trailing whitespace makes it my responsibility to reword.

Added a commit to redo this slightly

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**